src/sentry/web/frontend/debug/debug_chart_renderer.py

```diff
@@ -447,7 +447,185 @@
     "incidents": [incident],
     "selectedIncident": None,
 }
-
+metric_alert_with_anomalies = metric_alert.copy()
+metric_alert_with_anomalies["anomalies"] = [
+    {
```

Comment:
it's just a debug view so it's not critical but you can use these types https://github.com/getsentry/sentry/blob/master/src/sentry/seer/anomaly_detection/types.py#L5-L13

Reply:
actually I think `anomaly_type` in the types should be of type `AnomalyType`

Reply:
Oh oof.
this must've gotten lost in rebase.

will fix this